isobmff-byte-stream-format-respec.html

```diff
@@ -148,9 +148,9 @@
         (<span class="iso-box">elst</span>) that contains a single edit with media rate one. This edit may have a duration of 0 (indicating that it spans all subsequent media) or may have a non-zero duration
         (indicating the total duration of the movie including fragments).</p>
 
-      <p>The user agent must support parameter sets (e.g., PPS/SPS) stored in the sample entry (as defined for avc1/avc2), and should support parameter
-        sets stored inband in the samples themselves (as defined for avc3/avc4).</p>
-      <p class="note">For maximum content interoperability, user agents are strongly advised to support both inband and out-of-band storage of the SPS and PPS.</p>
+      <p>The user agent must support parameter sets (i.e., PPS/SPS) stored out-of-band in the sample entry, and should support parameter
```

Comment:
Should PPS/SPS be moved to the note too? Are those related to avc1/2? Perhaps restate some of this in the form of "For example, PPS/SPS in avc1/avc2..." (this is very rough).